### ISSUE TITLE
Example no longer works

### ISSUE THREAD

**Issue by Stephn-R** [vitalets/sinon-chrome#5]
I ran it on my machine after an npm install and got this

```bash
$ npm test

> sinon-chrome-example@0.1.0 test /Users/Stephen/Desktop/sinon-chrome/example
> phantomjs test/run.js

TypeError: 'undefined' is not a function (evaluating 'self.epilogue.bind(self)')

  ../node_modules/mocha/mocha.js:3980 in Spec
  ../node_modules/mocha/mocha.js:1822
  ../node_modules/mocha/mocha.js:6087
  test/run.js:21
```

**Comment by vitalets** [comment]
I will check, thanks!

**Comment by vitalets** [comment]
fixed in 0.2.1.

**Comment by forgetso** [comment]
Hi, I've followed these steps

`git clone https://vitalets/sinon-chrome
`cd sinon-chrome
`npm install
`npm install mocha
`npm install grunt
`npm install chai
`cd example
`npm test

and I get the same error as above

~/sinon-chrome/example 
(1:2123)$ npm test

> sinon-chrome-example@0.1.0 test /home/chris/sinon-chrome/example
> phantomjs test/run.js

TypeError: 'undefined' is not a function (evaluating 'self.epilogue.bind(self)')

  ../node_modules/mocha/mocha.js:4044 in Spec
  ../node_modules/mocha/mocha.js:1831
  ../node_modules/mocha/mocha.js:6556
  test/run.js:21
